src/games/strategy/triplea/TripleA.java

```diff
@@ -120,8 +120,13 @@ public void startGame(final IGame game, final Set<IGamePlayer> players, final bo
       } else {
         headlessFrameUI = null;
       }
-      display = new DummyTripleADisplay(headlessFrameUI);
-      soundChannel = new DummySoundChannel();
+      display = new TripleADisplay(mock(TripleAFrame.class)){
```

Comment:
It's safe to call the constructor with null instead of a mock since I'm overriding the used method